pybloqs/plot/core.py

```diff
@@ -411,7 +410,7 @@ def _write_plot(self, container, container_id, id_gen, static_output):
 
         self._write_plot_postprocess(chart_buf)
 
-        JScript.write_compressed(stream, chart_buf.getvalue())
+        JScript.write_compressed(stream, chart_buf.getvalue().encode())
```

Comment:
Should this be `.encode('utf-8')`? (I think the default encoding in Python 2 is `ascii`).